logos/tests/unit/main/test_v1_models.py

```
@@ -0,0 +1,221 @@
+    """Minimal DBManager stub used via monkeypatch."""
+
+    def __init__(self, models=None):
+        self._models = models if models is not None else []
+
+    def __enter__(self):
+        return self
+
+    def __exit__(self, *args):
+        return False
+
+    def get_models_for_profile(self, profile_id: int):
```

Ruff (0.15.1)

[warning] Unused method argument: `profile_id`

(ARG002)